app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -198,5 +185,26 @@ def track_events
       )
       track_billing_events
     end
+
+    def redirect_user(redirect_uri)
+      if IdentityConfig.store.openid_connect_redirect == :client_side
+        @oidc_redirect_uri = redirect_uri
+        render(
+          'openid_connect/shared/redirect',
+          layout: false,
+        )
+      elsif IdentityConfig.store.openid_connect_redirect == :client_side_js
+        @oidc_redirect_uri = redirect_uri
+        render(
+          'openid_connect/shared/redirect_js',
+          layout: false,
+        )
+      elsif IdentityConfig.store.openid_connect_redirect == :server_side
+        redirect_to(
+          redirect_uri,
+          allow_other_host: true,
+        )
+      end
```

Comment:
seems like a good fit for `case ... when`, and maybe we want to add a default case too?
```suggestion
      case IdentityConfig.store.openid_connect_redirect
      when :client_side
        @oidc_redirect_uri = redirect_uri
        render(
          'openid_connect/shared/redirect',
          layout: false,
        )
      when :client_side_js
        @oidc_redirect_uri = redirect_uri
        render(
          'openid_connect/shared/redirect_js',
          layout: false,
        )
      else # should only be :server_side
        redirect_to(
          redirect_uri,
          allow_other_host: true,
        )
      end
```